tests/internal/src/combine_static_library_test/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+ 
+################################
+cmake_minimum_required(VERSION 3.11)
+
+project(combine_static_library_test LANGUAGES C CXX)
+
+###############################################################################
+# Setup BLT
+###############################################################################
+# Set BLT_SOURCE_DIR to default location, if not set by user 
+if(NOT BLT_SOURCE_DIR)
+    set(BLT_SOURCE_DIR "${PROJECT_SOURCE_DIR}/../../../..")
+endif()
+
+include(${BLT_SOURCE_DIR}/SetupBLT.cmake)
+
+
+blt_add_library( NAME Foo1
+                 SOURCES Foo1.cpp
+                 INCLUDES Foo1.hpp
+               )
+           
+blt_add_library( NAME Foo23
+                 SOURCES Foo2.cpp Foo3.cpp
+                 INCLUDES Foo2.hpp Foo3.hpp
+               )
+           
+blt_combine_static_libraries( NAME FooStatic
```

Comment:
Are you sure these are actually being built in ``SHARED`` and ``STATIC`` mode?  See my comment above.

Reply:
yes